wandb/apis/public/runs.py

```diff
@@ -469,7 +548,8 @@ def load(self, force=False):
                 "projectId"
                 if _server_provides_internal_id_for_project(self.client)
                 else "",
-                RUN_FRAGMENT,
+                run_fragment.split()[1],  # Extract fragment name
```

Comment:
Don't do this, it will break if the string gets updated (for example, if someone were to add `#graphql` at the start for syntax highlighting). Just use `RunFragment` as before. Or we can introduce another constant `RUN_FRAGMENT_NAME` if we wanted to.